spec/javascripts/PaymentGateways/stripe/components/StripeCardForm.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { StripeElementsForm } from 'PaymentGateways/stripe/components/StripeFormWrapper'
+import { mount } from 'enzyme'
+
+const defaultProps = {
+    stripePublishableKey: 'abcd',
+    setupIntentSecret: 'efgh',
+    billingAddressDetails: {
+        line1: '1002 Avenue de los Mexicanos',
+        line2: '',
+        city: 'South Park',
+        state: 'CO',
+        postal_code: '80440',
+        country: 'US'
+    },
+    successUrl: '/Broflovski/Residence'
+}
+
+// NOTE: we mount StripeElementsForm instead of the StripeCardForm directly, because of the following requirement from Stripe:
+// Could not find Elements context; You need to wrap the part of your app that calls useStripe() in an <Elements> provider.
+
+it('should have cardholder name input', () => {
+    const wrapper = mount(<StripeElementsForm {...defaultProps} />)
+    expect(wrapper.find('.StripeElementsForm').exists()).toEqual(true)
+    const ccNameInput = wrapper.find('input#cardholder-name')
+    expect(ccNameInput.exists()).toBeTruthy()
```

Comment:
This assertion is redundant.

Reply:
Well, we first check whether the input exists, and then whether its placeholder has the expected text.
But I agree, we can only keep the second assertion.

Reply:
`find` should fail if the element does not exist.